huaweicloud/services/dew/resource_huaweicloud_kps_keypair.go

```diff
@@ -314,12 +314,19 @@ func parseEncodeValue(b []byte, err error) (string, error) {
 	return *rst, nil
 }
 
-func writeToPemFile(path, privateKey string) error {
-	var err error
+func writeToPemFile(path, privateKey string) (err error) {
```

Comment:
suggest to move writeToPemFile to util/common, it's duplicate with compute_keypair's

Reply:
OK, fix it in the next PR

Reply:
done